packages/grafana-ui/src/components/Table/styles.ts

```diff
@@ -52,14 +53,18 @@ export const getTableStyles = stylesFactory(
       headerCell: css`
         padding: ${padding}px 10px;
         cursor: pointer;
-        white-space: nowrap;
         color: ${colors.textBlue};
         border-right: 1px solid ${theme.colors.panelBg};
 
         &:last-child {
           border-right: none;
         }
       `,
+      headerCellLabel: css`
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+      `,
```

Comment:
I am sure you tried and this is needed, but can't this be on the headerCell class?

Reply:
unfortunatelly no :( and yes, I've tried.